content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,172 @@
+---
+reviewers:
+title: Role Based Access Control Good Practices
+description: >
+  Principles and practices for good RBAC design for cluster operators.
+content_type: concept
+weight: 10
```

Comment:
nit: Omit this.

The important outcome is that this page should sort _after_ “Controlling Access To The Kubernetes API” in the navigation.

Reply:
can I just remove it altogether, or just change the number?

Reply:
Remove it all together

Reply:
cool, done :)